src/main/webapp/i18n/de/tutorialGroupImport.json

```
@@ -6,19 +6,19 @@
             "importButton": "Übunsgruppen importieren",
             "dialogTitle": "Übungsgruppen importieren",
             "explanations": {
-                "description": "Dieser Dialog kann verwendet werden, um Übungsgruppen und deren Registrierungen aus einer CSV-Datei zu importieren. Dabei werden Übungsgruppen erstellt, wenn sie noch nicht existieren. Die Studierende werden von ihrer aktuellen Übungsgruppe abgemeldet und in der neuen Übungsgruppe registriert.",
-                "requirements": "Jede Zeile muss immer einen Wert für den Übungsgruppen Titel enthalten. Wenn nicht nur Übungsgruppen erstellt werden sollen, sondern auch die Registrierungen von Studierenden, muss die Zeile einen Wert für die Immatrikulationsnummer oder für den Benutzernamen enthalten. Dies ist erforderlich, um den Studierenden zu identifizieren. Werte für den Nachnamen und Vornamen sind optional. Die erste Zeile muss die Spaltennamen angeben!"
+                "description": "Dieses Dialogfeld ermöglicht es, Tutoriumsgruppen und Studentenregistrierungen aus einer CSV-Datei zu importieren. Wenn eine Tutoriumsgruppe noch nicht existiert, wird sie erstellt. Wenn Studenten in der CSV-Datei aufgeführt sind, werden sie von ihrer aktuellen Tutoriumsgruppe in die neue, in der CSV-Datei angegebene Gruppe verschoben.",
+                "requirements": "Jede Zeile muss den Titel der Tutoriumsgruppe enthalten. Um Studenten zu registrieren, muss jede Zeile entweder eine Matrikelnummer oder einen Login-Wert enthalten. Vor- und Nachname sind optional. Die oberste Zeile der CSV-Datei muss die Überschriften angeben."
             },
             "supportedHeaderNames": {
                 "label": "Unterstützte Spaltennamen (Versionen mit Leerzeichen, Unterstrichen, Bindestrichen und Grossbuchstaben werden ebenfalls unterstützt): ",
                 "alwaysRequired": "Immer erforderlich (um die Übungsgruppe zu identifizieren): ",
                 "requiredIfStudentRegistration": "Eines davon ist erforderlich, wenn Studierende registriert werden sollen (um Studierende zu identifizieren): ",
                 "alwaysOptional": "Immer optional (nur für Anzeigzwecke): ",
-                "tutorialGroupTitle": "Für den Tiel der Übungsgruppe: {{ headers }}",
-                "registrationNumber": "Für die Immatrikulationsnummer: {{ headers }}",
-                "login": "Für den Benutznermane: {{ headers }}",
-                "firstName": "Für den Vornamen: {{ headers }}",
-                "lastName": "Für den Nachnamen: {{ headers }}"
+                "tutorialGroupTitle": "Für den Tiel der Übungsgruppe: Übungsgruppe",
+                "registrationNumber": "Für die Immatrikulationsnummer: Matrikelnummer",
+                "login": "Für den Benutznermane: Benutzername",
```

```suggestion
                "login": "Für den Benutzernamen: Benutzername",
```
- typo on the left side of the colon
- Should be ‘Login’ instead according to guidelines https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines.html#german-language
  - Can this be changed or does this reference the name of the CSV column as well and would change the required file format if changed in code to match the other name?

---

I think this should be replaced with `Benutzendennamen` (or, if you think this sounds weird, `Anmeldename`) to be gender-neutral and fulfill the guidelines too.

You could also reuse `Login Name` as suggested by @b-fein .

---

changed to Anmeldename 👍